File: MyCoffeeShop/app/src/main/java/com/example/mycoffeeshop/MainActivity.java
package com.example.mycoffeeshop;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    LinearLayout menu;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        menu = (LinearLayout) findViewById(R.id.parentMenu);
        menu();
    }

    private void menu(){
        for (int i = 0; i<menu.getChildCount();i++){
            CardView menuItem = (CardView) menu.getChildAt(i);
            if (i == 0){
                menuItem.setOnClickListener(new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                        Intent intent = new Intent(getApplicationContext(), list_order.class);
                        startActivity(intent);
                    }
                });
            }
            else if(i == 1){
                menuItem.setOnClickListener(new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                        Intent intent = new Intent(getApplicationContext(), list_barang.class);
                        startActivity(intent);
                    }
                });
            }
            else if(i == 2){
                menuItem.setOnClickListener(new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                        Toast.makeText(getApplicationContext(), "Cardview ke-3", Toast.LENGTH_SHORT).show();
                    }
                });
            }
        }
    }
}

File: LoginUI/app/src/main/java/com/example/loginui/MainActivity.kt
package com.example.loginui

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.example.loginui.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    private lateinit var viewRoot: View
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        viewRoot = binding.root
        setContentView(viewRoot)
        binding.btnLogin.setOnClickListener(View.OnClickListener {
            val username = binding.edUsername.text.trim().toString()
            val password: String = binding.edPassword.text.trim().toString()
            if (username.contentEquals("admin") && password.contentEquals("admin")) {
                val intent = Intent(this, HomeScreen::class.java)
                startActivity(intent)
                finish()
            }
            else {
                Toast.makeText(this, "Username atau Password Anda tidak benar!", Toast.LENGTH_SHORT)
                    .show()
            }
        })
    }
}